packages/client/hmi-client/src/types/Types.ts

```diff
@@ -554,9 +554,11 @@ export interface CalibrationRequestCiemss {
 }
 
 export interface CiemssStatusUpdate {
-    loss: number;
-    progress: number;
     jobId: string;
+    progress: number;
+    loss?: number;
+    currentResults?: number[];
+    totalPossibleIterations?: number;
```

Comment:
Any particular reason we have the same update-object servicing different operations?  
Is it too much problem to have them separate? It is a bit awkward because as a new reader you don't really know which fields should go with which other fields.

Reply:
I will look to change this into two classes

Reply:
Ive split this into two classes, this involved updating how we parse these in `SimulationEventService`.
After speaking with Kevin this seems like a reasonable approach for this